cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java

```diff
@@ -0,0 +1,162 @@
+package com.powsybl.cgmes.conversion;
```

Comment:
Add Copyright. Could you also add copyrights to `RegulatingControlMappingForGenerators` and `RegulatingControlMappingForTransformers`?

Reply:
Done